gcc/rust/util/rust-hir-map.cc

```diff
@@ -1241,6 +1241,9 @@ Mappings::lookup_builtin_marker ()
   return builtinMarker;
 }
 
+// FIXME: Before merging: Should we remove the `locus` parameter here? since
+// lang items are looked up mostly for code generation, it doesn't make sense to
+// error out on the locus of the node trying to access an inexistant lang item
```

Comment:
This would require changes in a lot of code which fetches lang items, such as the typechecker and codegen. It would be done in a later PR